client/search-ui/src/input/extensions/completion.ts

```diff
@@ -146,16 +146,30 @@ export function searchQueryAutocompletion(
         // This renders the completion icon
         {
             render(completion) {
-                return createSVGIcon(
+                const icon = createSVGIcon(
                     completion.type && completion.type in typeIconMap
                         ? typeIconMap[completion.type as CompletionType]
                         : typeIconMap[SymbolKind.UNKNOWN]
                 )
+                if (completion.type && completion.type in typeIconMap) {
+                    icon.setAttribute('aria-label', completion.type)
+                }
```

Comment:
If there is no label, the icon should be hidden.

```suggestion
                if (completion.type && completion.type in typeIconMap) {
                    icon.setAttribute('aria-label', completion.type)
                } else {
                    icon.setAttribute('aria-hidden', 'true')
                }
```

FWIW no icon seems to have labels when I checked this out to test it? 
![image](https://user-images.githubusercontent.com/206864/200438597-7e954b17-416b-438b-ad5a-de2a7eaea78c.png)